experimenter/experimenter/nimbus_ui_new/static/js/scripts/experiment_detail.js

```diff
@@ -0,0 +1,14 @@
+function toggleJSONView(showFull) {
+  const previewDiv = document.getElementById("recipe-preview");
+  const fullDiv = document.getElementById("full-recipe-json");
+
+  if (showFull) {
+    previewDiv.style.display = "none";
+    fullDiv.style.display = "block";
+  } else {
+    fullDiv.style.display = "none";
+    previewDiv.style.display = "block";
+  }
+}
+
+window.toggleJSONView = toggleJSONView;
```

Comment:
Oh no you should be able to do this without needing any custom javascript using the bootstrap collapse component:

https://getbootstrap.com/docs/5.3/components/collapse/

Reply:
Okay @yashikakhurana is out so I'll take care of this.